package-lock.json

```diff
@@ -1,15360 +1,14 @@
 {
   "name": "stylelint-config-standard",
   "version": "21.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Oh, the lockfile version is downgraded! 😅

Reply:
Oh, so sorry - missed that in the diff. Will update right now 😅